tests/legacy-cli/e2e/tests/misc/third-party-decorators.ts

```diff
@@ -6,10 +6,13 @@ import { updateJsonFile } from '../../utils/project';
 export default async function () {
   await updateJsonFile('package.json', (packageJson) => {
     // Install ngrx
-    packageJson['dependencies']['@ngrx/effects'] = '^9.1.0';
-    packageJson['dependencies']['@ngrx/schematics'] = '^9.1.0';
-    packageJson['dependencies']['@ngrx/store'] = '^9.1.0';
-    packageJson['dependencies']['@ngrx/store-devtools'] = '^9.1.0';
+    packageJson['dependencies']['@ngrx/effects'] = '^13.2.0';
+    packageJson['dependencies']['@ngrx/schematics'] = '^13.2.0';
+    packageJson['dependencies']['@ngrx/store'] = '^13.2.0';
+    packageJson['dependencies']['@ngrx/store-devtools'] = '^13.2.0';
+
+    // TODO(crisbeto): ngrx hasn't been updated for TS 4.7 yet.
+    packageJson['devDependencies']['typescript'] = '~4.6.2';
```

Comment:
nit: With the default of TS 4.6, this shouldn't be needed now.  Hopefully ngrx will be updated soon with support.

Reply:
I did this under the assumption that we do go forward with 4.7.